cmd/juju/status/formatted.go

```diff
@@ -148,6 +150,39 @@ func (s remoteApplicationStatus) MarshalYAML() (interface{}, error) {
 	return remoteApplicationStatusNoMarshal(s), nil
 }
 
+type offerStatusNoMarshal offerStatus
```

Comment:
Oh, is it because it's not embedded then it hides the `Marshal*` methods? Expanding a bit I can see it's all through the package, so I guess that's just how we do this?